demo/src/com/google/maps/android/utils/demo/HeatmapsPlacesDemoActivity.java

```diff
@@ -79,7 +79,7 @@
     /**
      * Places API server key.
      */
-    private static final String API_KEY = "YOUR_KEY_HERE"; // TODO place your own here!
+    private static final String API_KEY = "AIzaSyDe3Ivf9_tRPzoxzSKIWxJxH6uCLB6H9ww"; // TODO place your own here!
```

Comment:
Revert this. It really should live in a resource file but that's not in scope for your changes.